spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialFullConvolution.scala

```diff
@@ -75,7 +76,9 @@ class SpatialFullConvolution[A <: Activity : ClassTag, T: ClassTag](
   var adjH: Int = 0,
   val nGroup: Int = 1,
   val noBias: Boolean = false,
-  private var initMethod: InitializationMethod = Default
+  private var initMethod: InitializationMethod = Default,
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out?